pynetbox/api.py

```diff
@@ -16,6 +16,7 @@
 from pynetbox.core.endpoint import Endpoint
 from pynetbox.core.query import Request
 from pynetbox.models import dcim, ipam, virtualization, circuits
+import requests
```

Comment:
I think the heavy lifting in this fits a bit better in `query.py`, plus we're already brining in `requests` there.

Reply:
Fair point. I created Request.get_version(), is that the way of doing it in query.py?

Reply:
Yep, perfect.